user-service/src/main/resources/application.yaml

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 #
 spring:
+  cache:
+    caffeine:
+      spec: expireAfterWrite=60s, maximumSize=500
```

Comment:
If this is the default config then doesn't this mean that if you have more than 60 seconds between configuring the example and running it, then you will hit user not found exceptions as we are using the null-user-service

Reply:
If my above assumption is correct then I suggest making the ttl 1hour

Reply:
agreed to set ttl to 60m